tools/trusted-app-client/src/main/java/edu/emory/cci/bindaas/trusted_app_client/app/BindaasTrustedClientApp.java

```diff
@@ -362,7 +394,24 @@ private Arguments parseRevokeArguments(CommandLine line) throws IllegalArgumentE
 		{
 			throw new IllegalArgumentException("[username] not specified");
 		}
-		
+
+		if(line.hasOption("protocol"))
+		{
+			if(line.getOptionValue("protocol").matches("api_key|jwt")){
+				args.protocol = line.getOptionValue("protocol");
+			}
+			else {
+				throw new IllegalArgumentException("Illegal value for [protocol]");
+			}
+
+		}
+		else
+		{
+			args.protocol = "api_key";
+			System.out.println("[protocol] not specified");
```

Comment:
replace 
System.out.println() with log.info()

here and everywhere else. 

Do not introduce System.out.println() to code. If you find some, replace them with log.info() as you find them.

Reply:
trusted-app-client does not have the logging module. I will add that and replace it everywhere.

Reply:
Yes, pls add it. 

From maven (pom.xml) perspective:
Since now we are integrating the trusted-app-client into Bindaas main build, we can even import it from the parent/root pom.xml.